torch/csrc/jit/codegen/cuda/scheduler/registry.cpp

```diff
@@ -857,6 +903,14 @@ class PointWiseScheduler : public SchedulerEntry {
   }
 
   static bool canScheduleCompileTime(Fusion* fusion) {
+    // TODO: extend this for view support
+    //   Currently using the same path as the scheduler
+    // to eliminate mismatch between canSchedule and
+    // schedule pointwise.
+    if (!hasReferenceTensorView(fusion)) {
```

Comment:
@csarofeen we might want to use the same code path for `canSchedule` and `schedule` to ensure not hitting similar errors. This will unblock the current issue. Could we use this as an intermediate state and later maybe build out on `DomainMap` for view support?

Reply:
I think the reference check correctly handles view, I forgot we even had that logic, thanks for tracking it down. I don't know if there's anything else we need to worry about. Is there another case that will fail today?

Reply:
I see. The reference check looks fine for now. Didn't see more failure cases. Updated comment to remove "TODO".